planetarypraxis/smartforests
# fix: don't display untranslated logbooks on the index page

## Description
When the Portugese locale was created, aliases of the English pages were created and added to the Portugese site. This meant that English content was displayed in the Portugese site, in particular on the logbooks page. These untranslated aliases are now filtered out so the logbooks page only displays Portugese content.

## Motivation and Context
This solves the issue of English content being displayed on the Portugese page. The issue is https://linear.app/commonknowledge/issue/SF3-60/inconsistency-of-translation.

## How Can It Be Tested?
The easiest way to test is to copy the live database and make sure the Portugese logbooks index page doesn't display English content.

## How Will This Be Deployed?
Standard deploy.

## Types of changes


## Checklist:
- My change requires a change to the documentation.
- I've updated the documentation accordingly.
